script/build

```diff
@@ -268,11 +268,24 @@ def main() -> None:
                 *cmake_args,
                 verbose=True,
             )
-        except subprocess.CalledProcessError as exc:
+        except subprocess.CalledProcessError:
             print_error("Build failed. If the output makes no sense, try these steps:")
             print_error("1. Delete '_skbuild' directory")
             print_error(f"2. Delete conan directory: {Path.home() / '.conan'}")
             sys.exit(1)
+        setup_scripts = ["src/clib/lib/job_queue/qstat_proxy.sh"]
```

Comment:
I sadly didn't manage to find a smart way of fetching this list from `setup.py` : (

Reply:
no longer relevant, resolving